### PR TITLE
Fix transaction amount overflow

### PR DESCRIPTION
**Pull request by nymkappa**
Fixes https://github.com/mempool/mempool/issues/3551
See https://github.com/mempool/mempool/pull/3571#issuecomment-1483755270

### CODE REVIEW

**Comment by wiz** [comment]
how about this one https://liquid.network/testnet/tx/5dd70e2f7c0f06c514fa5daf9a19d1b2961ae91132d8d2a88dea38f102ffe049

**Comment by nymkappa** [comment]
> how about this one https://liquid.network/testnet/tx/5dd70e2f7c0f06c514fa5daf9a19d1b2961ae91132d8d2a88dea38f102ffe049

Flip your phone

**Comment by nymkappa** [comment]
> how about this one https://liquid.network/testnet/tx/5dd70e2f7c0f06c514fa5daf9a19d1b2961ae91132d8d2a88dea38f102ffe049

Screenshot 2023-03-25 at 16 40 31
Screenshot 2023-03-25 at 16 40 20
Screenshot 2023-03-25 at 16 37 21
Screenshot 2023-03-25 at 16 37 06

File: frontend/src/app/components/amount/amount.component.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export class AmountComponent implements OnInit, OnDestroy {
   currencySubscription: Subscription;
 
   @Input() satoshis: number;
-  @Input() digitsInfo = '1.8-8';
+  @Input() digitsInfo = '1.0-8';
```
Comment (softsimon) [inline frontend/src/app/components/amount/amount.component.ts:22]: I think we need to really think through this change.

I don't like how some Bitcoin services remove the zeroes make it difficult to see how many satoshis it actually was.. for example

`0.0000001`

Is this 1 satoshi or 10 satoshis?

Comment (wiz) [reply frontend/src/app/components/amount/amount.component.ts:22]: yes must always show 8 digits

Comment (nymkappa) [reply frontend/src/app/components/amount/amount.component.ts:22]: I think it's not even necessary anymore. Was from a previous commit. Will remove.

**Review by softsimon — CHANGES_REQUESTED** [review]
I think it's too much of a width reduction.

Screenshot 2023-07-20 at 14 43 46

Screenshot 2023-07-20 at 14 44 21

**Comment by nymkappa** [comment]
@softsimon I've updated the min sats amount required to apply the aggressive truncate, see examples:

Screenshot 2023-08-05 at 10 51 45
Screenshot 2023-08-05 at 10 51 30
Screenshot 2023-08-05 at 10 49 25

**Review by softsimon — APPROVED** [review]
tested ACK @ [683b7238]

**Pull request merged**